config/default.yml

```diff
@@ -656,6 +656,13 @@ Rails/MigrationClassName:
   Include:
     - db/**/*.rb
 
+Rails/MigrationTimestamp:
+  Description: 'Checks that migration filenames start with a valid timestamp.'
+  Enabled: pending
+  VersionAdded: '<<next>>'
+  Include:
+    - db/migrate/**/*.rb
```

Comment:
I noticed that `MigrationClassName` opts to include everything under `db`, and uses a node pattern to detect if the file looks like a migration. We could probably do something similar here, if that's preferable.

Reply:
> if the file looks like a migration

I've wondered about that.
What does "look like a migration" mean really? (We can't rely on the filename having a timestamp prefix, considering that's what we want to check here.)
Would you open the migration file and check if it defines a class that inherits from `ActiveRecord::Migration`? What if a project implements its own `ApplicationMigration` base class?

Reply:
> Would you open the migration file and check if it defines a class that inherits from `ActiveRecord::Migration`?

That is [_exactly_](https://github.com/rubocop/rubocop-rails/blob/master/lib/rubocop/cop/mixin/migrations_helper.rb) what it does! 😅